content/use-cases/defi/algorithmic-trading.md

```diff
@@ -0,0 +1,89 @@
+---
+html: algorithmic-trading.html
+parent: defi-uc.html
+blurb: The XRP Ledger's decentralized exchange consists of an unlimited number of currency pairs, tracked on-demand when users make trades. 
+labels:
+  - Transactions
+---
+# Algorithmic Trading
+
+The XRP Ledger's decentralized exchange presents an opportunity to earn money through _algorithmic trading_, which means running a computer program to find and take profitable trading opportunities automatically. In algorithmic trading, you typically make many trades based on quantitative factors to earn steady, small profits; this is unlike traditional manual trading where you make a few long-term investments based on market fundamentals and wait to earn a large return over time. Blockchains are often more suitable for algorithmic trading than manual trading, because the high volatility of cryptocurrencies in general makes them less suitable for traditional "buy and hold" investing; the XRP Ledger is particularly suited for algorithmic trading, for several reasons:
+
+- Decentralized exchange data is public and freely available. You don't need a subscription or special access to see what trades are available.
+- Trades settle in seconds, which is fast enough that you can do a lot of trading, but not so fast that you need a specialized network connection or extremely powerful computer. You can trade using a laptop and decent home internet connection.
```

Comment:
HFT is typically done on the level of microseconds, so that's why I didn't use that term here. Rephrased instead to:

> Trades settle in seconds, enabling frequent trading without specialized equipment.